config/eloquent-driver.php

```diff
@@ -42,6 +41,11 @@
         'entry' => \Statamic\Eloquent\Entries\Entry::class,
     ],
 
+    'fieldsets' => [
+        'driver' => 'eloquent',
```

Comment:
This should be `file` by default, like the other repositories.

```suggestion
        'driver' => 'file',
```

Otherwise, as soon as you install the Eloquent Driver, it'll attempt to get fieldsets from the database even if you haven't migrated the table, causing an error.

Reply:
Good spot, forgot to change it when testing